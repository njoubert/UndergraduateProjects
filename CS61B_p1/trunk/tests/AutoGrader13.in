Dmitry does not live in the russian house.
The physicist does not live in the russian house.
Dmitry is not the chemist.
Wolodja is not the physicist.
Roman is not the diplomat.
The diplomat does not live in the belorussian house.
The chemist does not live in the ukrainian house.
Roman does not live in the belorussian house.
What do you know about Wolodja?
What do you know about the physicist?
What do you know about the chemist?
Where does the chemist live?
Who is the diplomat?
Where does Dmitry live?
Who lives in the russian house?

