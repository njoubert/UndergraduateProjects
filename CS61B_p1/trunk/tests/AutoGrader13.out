1. Dmitry does not live in the russian house.
2. The physicist does not live in the russian house.
3. Dmitry is not the chemist.
4. Wolodja is not the physicist.
5. Roman is not the diplomat.
6. The diplomat does not live in the belorussian house.
7. The chemist does not live in the ukrainian house.
8. Roman does not live in the belorussian house.
Q: What do you know about Wolodja?
A: Wolodja is the diplomat and lives in the ukrainian house.
Q: What do you know about the physicist?
A: Dmitry is the physicist and lives in the belorussian house.
Q: What do you know about the chemist?
A: Roman is the chemist and lives in the russian house.
Q: Where does the chemist live?
A: The chemist lives in the russian house.
Q: Who is the diplomat?
A: Wolodja is the diplomat.
Q: Where does Dmitry live?
A: Dmitry lives in the belorussian house.
Q: Who lives in the russian house?
A: Roman lives in the russian house.
